contrib/common.sh

```diff
@@ -94,6 +94,14 @@ function configure_search() {
   rake -f /vagrant/src/api/Rakefile ts:rebuild
 }
 
+function setup_signd() {
+  echo "Setting up signd so bs_signer can be used..."
+  cd / && tar xf /vagrant/dist/obs-signd-conf.tar.bz2
+  systemctl enable rng-tools
+  systemctl start rng-tools
```

Comment:
maybe it is safer to use haveged instead, since rng-tools may fail on systems without hardware random generator.